tests/unit/test_docs.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for the documentation."""
+import collections
+import os
+import re
+from typing import List
+
+# For entries in the following list, two appearances (but not more) are
+# allowed. This can be useful if a pull request needs to appear in multiple
+# sections.
+ALLOWED_EXCEPTIONS: List[str] = []
```

Comment:
quite baffled by the need for this variable :confused:

Reply:
See discussion here: ESMValGroup/ESMValTool#2554. I thought it might be good to add this feature to this test as well if we need to allow duplicates at some point.